client/src/components/App/App.jsx

```diff
@@ -1,19 +1,31 @@
 import styled from 'styled-components'
-import Header from 'components/Header/Header'
+import quarterCircle from 'assets/quarterCircle.svg'
 import { recommendedWines } from 'data/wine_recs_mixed_small.json'
+import Header from 'components/Header/Header'
 import WineListing from 'components/WineListing/WineListing'
-import quarterCircle from 'assets/quarterCircle.svg'
-import { useSaveWine } from 'lib/customHooks'
+import WineStorage from 'components/WineStorage/WineStorage'
+import { Route, Switch } from 'react-router-dom'
+import useHandleSave from 'hooks/useHandleSave'
 
 export default function App() {
-  const [savedWines, { handleSaveWine }] = useSaveWine()
-  console.log(savedWines)
+  const [savedWines, toggleSave] = useHandleSave()
 
   return (
     <Grid>
       <Header title="Vinz" subtitle="Wine Assistant and Taste Journal" />
       <Main>
-        <WineListing results={recommendedWines} onBookmark={handleSaveWine} />
+        <Switch>
+          <Route path="/wine-storage">
+            <WineStorage savedWines={savedWines} onSaveToggle={toggleSave} />
+          </Route>
+          <Route path="/">
```

Comment:
Just in case, I think I'd add an `exact` if the `WineListing` component should only be rendered then. I know it's not necessary as you put it at last route in the switch, but you never know :) just my thoughts though.
```suggestion
          <Route exact path="/">
```